trillian/integration/ct_hammer/main.go

```diff
@@ -194,10 +194,16 @@ func main() {
 		if err != nil {
 			glog.Exitf("Failed to create client pool: %v", err)
 		}
+
+		mmd := *mmd
+		if c.MaxMergeDelaySec != 0 {
+			mmd = time.Second * time.Duration(c.MaxMergeDelaySec)
```

Comment:
The combination of this and the test config values will disable the hammer's ability to do get-proof-by-hash, so we need to think about this a bit more carefully.  

The hammer only attempts get-proof-by-hash if its idea of the MMD for a submitted cert has passed, and this PR changes that value from 2 minutes to 24 hours.  You can see the problem in your Travis hammer output:
```
 athos: lastSTH.size=1209 ops: total=1002 invalid=118 errs=0 AddChain=374/374 AddPreChain=445/445 GetSTH=45/45 GetSTHConsistency=38/41 GetProofByHash=0/42 
```
which shows no GetProofByHash operations happening.

So maybe:
 - use the effective merge delay in the hammer, with a comment about this not being a guarantee but should be OK 
 - lower the effective MMD in the config files to a few minutes rather than 2 hours (with a comment next to it saying this is for testing, too low for a real log)

Reply:
Done.
Btw, shouldn't this output in Travis have resulted in a pre-submit failure?